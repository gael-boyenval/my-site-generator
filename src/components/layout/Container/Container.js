import styled from 'styled-components';
import { unit, spaces } from 'Styles/tokens';
import { rem } from 'Styles/utils';

const Container = styled.div`
  max-width: ${rem(unit * 108)};
  margin: 0 auto;
  padding: 0 ${rem(spaces.xl)};
`;

export default Container;
