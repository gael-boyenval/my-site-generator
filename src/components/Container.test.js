/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import 'jest-styled-components';

import Container from './Container';

configure({ adapter: new Adapter() });

describe('Component Container', () => {
  it('should be render without crash', () => {
    const component = renderer.create(<Container>Hello</Container>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
