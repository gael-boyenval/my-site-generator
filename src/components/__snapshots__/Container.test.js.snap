// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Container should be render without crash 1`] = `
<div
  style={
    Object {
      "margin": "3rem auto",
      "maxWidth": 600,
    }
  }
>
  zetfrqjzegfwhxfkjwshfgkjsdhfgqk jsdhfg qsjkdfhg q
  Hello
</div>
`;
